docs/user/index.md

```diff
@@ -1,33 +1,27 @@
 # Usage Documentation
 
+## Starting with Cookbook
 
-## Importing Recipes
+Before creating your first recipe, you should decide where your recipes and pictures will be stored.
+All files will be accessible in your Nextcloud file manager.
 
-### Importing from a Website
+- Recipe's directory: check in the *Parameters* of Cookbook, in the low left corner, in which directory they will be stored on Nextcloud.
+- Pictures directory: it's also a good idea to create a directory to gather all Cookbook pictures in one place.
```

Comment:
May I ask you @zorglubu what the intention of a dedicated image folder would be? From the perspective of the cookbook app it is definitively not required. Contrary, this will store all recipe images twice: Once in the common folder and once per recipe.

I do not want to silently remove your wording. So, I am just curiously asking for your opinion/reason for this suggestion. Feel free to communicate. This might seem a general misunderstanding and thus a good point for documentation :smile:

Reply:
That clearly means that I didn't understand something, and that a clear documentation would be a good thing  ;-)
Fell free to modify that, and by the way, I will be grateful if you check my english everywhere !

As a new user, I did put the pictures in a separate folder like  /photos/recipes, and I just now discovered that they were as well in the recipe's sub-folder. But when we create a new recipe, this sub-folder doesn't exist yet. Then I presume that the process should be :
- to upload this picture somewhere in Nextcloud
- to add it to the recipe
- and finally to delete the first one ?

Reply:
There are different ways.
- One, you have described.
- Use an external URL if you have the image on the web
- There is I think some heuristics in place ATM to look for a file within the NC server files
- There is a plan for adding an image on the fly. We could then enhance the UI to upload an image directly.